gui/gm-unit.lua

```diff
@@ -613,6 +613,700 @@ function editor_wounds:init( args )
 end
 add_editor(editor_wounds)
 
+-- ATKANA EDIT START
+local setbelief = dfhack.reqscript("modtools/set-belief")
+local setpersonality = dfhack.reqscript("modtools/set-personality")
+local setneed = dfhack.reqscript("modtools/set-need")
+local setorientation = dfhack.reqscript("set-orientation")
```

Comment:
any reason this is not same as others? (i.e. modtools folder)

Reply:
I didn't include set-orientation as a modtool basically as a whim (and I don't fully know what counts as a modtool and what doesn't). If it seems like it should belong as a modtool (or that the others shouldn't), somebody more knowledgable about them is welcome to change them.

Reply:
Generally if they are end-user-facing they shouldn't be in modtools (my rule of thumb is "end users shouldn't have to type 'modtools' into the DFHack console). Looks like most of these are at least somewhat user-facing, so I would probably move them out.